airflow/www/static/js/task.js

```diff
@@ -17,11 +17,18 @@
  * under the License.
  */
 
-/* global document */
-import { formatDateTime } from './datetime_utils';
+/* global document, moment */
 
-// reformat execution date to be more human-readable
+// reformat task details to be more human-readable
 document.addEventListener('DOMContentLoaded', () => {
-  const date = document.getElementById('ti_execution_date');
-  date.innerHTML = formatDateTime(date.innerHTML);
+  document.querySelectorAll('.ti-attr').forEach((attr) => {
```

Comment:
Sorry I didn't catch this last time, but should we change this one as well?
```suggestion
  document.querySelectorAll('.js-ti-attr').forEach((attr) => {
```